tests/test_sealed_box.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2013 Donald Stufft and individual contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import, division, print_function
+
+import binascii
+
+import pytest
+
+from nacl.encoding import HexEncoder
+from nacl.public import PrivateKey, PublicKey, SealedBox
+
+
+VECTORS = [
+    # privalice, pubalice, plaintext, encrypted
+    (
+        b"77076d0a7318a57d3c16c17251b26645df4c2f87ebc0992ab177fba51db92c2a",
```

Comment:
I just found I've added a relevant comment in https://gist.github.com/lmctv/d76a2ebefd3227c461ff5450ea487a11 
```
    # libsodium's test don't provide reference data for
    # the sealed box construction.
    # the following has been manually generated
```
In asmuch as I can remember, I've "played the API" on the command line, then verified the python + SecretBox implementation in the gist could correctly decrypt the manually generated data, and kept that data as a reference for cffi implementation.

This is the main reason why I do a round-trip test in test_sealed_box_encryption.